target/linux/pistachio/dts/pistachio_marduk_common.dtsi

```diff
@@ -280,6 +280,11 @@
 &i2c2 {
 	status = "okay";
 	clock-frequency = <400000>;
+	tpm_i2c_slb9645@20 {
```

Comment:
would be good to have dts change in a separate commit so that we can mention explicitly

pistachio_marduk_common.dtsi: add tpm config or something like that
Mostly we have been following to mention "dts" or "creator-platform-default configs" filenames/module names in the commit messages.